utils/generateTokensArray.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+
+const tokenGroupsDir = path.join(__dirname, '../src/tokens/token-groups');
+
+const generateTokensArray = () => {
```

Comment:
Awesome!

**suggestion (non-blocking):** Small nit but stylistically I try to avoid using data type names in vars and functions. What do you think about `getCustomProperties()` or `getCustomPropertyNames()`. I kind of like the latter (`getCustomPropertyNames`) as `getCustomProperties` could be confused for key/value pairs.